src/systems/velocity_demo/VelocityDemo.hh

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef IGNITION_GAZEBO_SYSTEMS_VELOCITYDEMO_HH_
+#define IGNITION_GAZEBO_SYSTEMS_VELOCITYDEMO_HH_
+
+#include <ignition/gazebo/System.hh>
+#include <memory>
+
+namespace ignition
+{
+namespace gazebo
+{
+// Inline bracket to help doxygen filtering.
+inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
+namespace systems
+{
+  // Forward declaration
+  class VelocityDemoPrivate;
+
+  /// \brief Differential drive controller which can be attached to a model
+  /// with any number of left and right wheels.
```

Comment:
Mind updating this system's documentation?

Also, I suggest not calling it a demo, since it is installed and it should be useful for common usage, right? I'm not sure what a good name would be. I'm actually wondering if maybe this should be combined with `DiffDrive`.

Reply:
how about `VelocityControl`?